code/modules/mob/mob.dm

```diff
@@ -487,6 +487,7 @@ var/list/slot_equipment_priority = list( \
 		return 0 //Unsupported slot
 		//END HUMAN
 
+// If you're looking for `reset_perspective`, that's a synonym for this proc.
```

Comment:
Do we have any strong reasons to continue using reset_view over reset_perspective? Considering the amount of features we port from /tg/, we may as well start using their proc name.